modules/core/nuxeo-core-api/src/test/resources/OSGI-INF/dummy-blob-provider.xml

```diff
@@ -12,5 +12,15 @@
     <blobprovider name="dummy">
       <class>org.nuxeo.ecm.core.blob.DummyBlobProvider</class>
     </blobprovider>
+    <blobprovider name="fromdoc">
+      <class>org.nuxeo.ecm.core.blob.DummyBlobProvider</class>
+    </blobprovider>
+    <blobprovider name="fromdoc2">
+      <class>org.nuxeo.ecm.core.blob.DummyBlobProvider</class>
```

Comment:
Should we configure it as allowing byte range?

Reply:
No this one is just a "virtual" blob provider, it doesn't need byte ranges. It's the one the virtual implementation is actually using that needs byte ranges.